tests/unit/accordion/accordion_core.js

```diff
@@ -7,15 +7,23 @@ module( "accordion: core", setupTeardown() );
 
 $.each( { div: "#list1", ul: "#navigation", dl: "#accordion-dl" }, function( type, selector ) {
 	test( "markup structure: " + type, function() {
```

Comment:
So your saying you think i should remove the theme classes and test only the structure here?